server/task_executor.go

```diff
@@ -243,7 +243,17 @@ func postMigration(ctx context.Context, server *Server, task *api.Task, vcsPushE
 			bytebaseURL = fmt.Sprintf("%s/issue/%s?stage=%d", server.profile.ExternalURL, api.IssueSlug(issue), task.StageID)
 		}
 
-		commitID, err := writeBackLatestSchema(ctx, server, repo, vcsPushEvent, mi, repo.BranchFilter, latestSchemaFile, schema, bytebaseURL)
+		// TODO(d): we need to figure out the baseline write-back for users using wildcard branch filter.
+		branch := repo.BaseDirectory
```

Comment:
this should be branchFilter

Reply:
LOL, nice catch!

Reply:
Nice catch, fixed: https://github.com/bytebase/bytebase/pull/3178